addons/html_editor/static/tests/table/adding_table.test.js

```diff
@@ -108,3 +108,50 @@ test.tags("iframe", "desktop")(
         expect(":iframe .o_table").toHaveCount(1);
     }
 );
+
+test.tags("desktop")("Expand columns in the correct direction in 'rtl'", async () => {
+    const { editor } = await setupEditor("<p>a[]</p>", {
+        config: {
+            direction: "rtl",
+        },
+    });
+    insertText(editor, "/");
+    await waitFor(".o-we-powerbox");
+    insertText(editor, "table");
+    await animationFrame();
+    press("Enter");
+    await waitFor(".o-we-tablepicker");
+
+    // Initially we have 3 columns
+    expect(".o-we-tablepicker").toHaveCount(1);
+    const tablePickerOverlay = document.querySelector(".overlay");
+    expect(tablePickerOverlay).toHaveStyle({ right: /px$/ });
+    const right = tablePickerOverlay.style.right;
+    const width3Columns = tablePickerOverlay.getBoundingClientRect().width;
+    expect(".o-we-cell.active").toHaveCount(9);
+
+    // Add one column -> we have 4 columns
+    press("ArrowLeft");
+    await animationFrame();
+    expect(tablePickerOverlay.getBoundingClientRect().width).toBeGreaterThan(width3Columns);
```

Comment:
I m not sure to understand what you want to test ?

Reply:
Maybe it's not realy needed, I'm trying to make sure that the width actually increased/decreased when we add/remove columns